turbo.json

```diff
@@ -18,7 +19,6 @@
     "lint": {},
     "check-types": {},
     "dev": {
-      "dependsOn": ["^build"],
```

Comment:
Why did you remove this? that used to be what made the glue work.

Reply:
Not relevant for an internal dependency